doc/releases/quincy.rst

```diff
@@ -0,0 +1,3627 @@
+  scrub begins.
+
+* A health warning is now reported if the ``require-osd-release`` flag
+  is not set to the appropriate release after a cluster upgrade.
+
+* LevelDB support has been removed. ``WITH_LEVELDB`` is no longer a supported
+  build option.
+
+* Cephadm: ``osd_memory_target_autotune`` is enabled by default, which sets
+  ``mgr/cephadm/autotune_memory_target_ratio`` to ``0.7`` of total RAM. This
+  is unsuitable for hyperconverged infrastructures. For hyperconverged Ceph,
+  please refer to the documentation or set
+  ``mgr/cephadm/autotune_memory_target_ratio`` to ``0.2``.
+
+* telemetry: Improved the opt-in flow so that users can keep sharing the same
+  data, even when new data collections are available. A new 'perf' channel that
+  collects various performance metrics is now avaiable to opt into with:
+  `ceph telemetry on`
+  `ceph telemetry enable channel perf`
+  See a sample report with `ceph telemetry preview`.
+  Note that generating a telemetry report with 'perf' channel data might
+  take a few moments in big clusters.
+  For more details, see:
+  https://docs.ceph.com/en/quincy/mgr/telemetry/
+
+* MGR: The progress module disables the pg recovery event by default since the
+  event is expensive and has interrupted other services when there are OSDs
+  being marked in/out from the the cluster. However, the user can still enable
+  this event anytime. For more detail, see:
+
+  https://docs.ceph.com/en/quincy/mgr/progress/
+
+Dashboard
+~~~~~~~~~
+* Day 1: the new "Cluster Expansion Wizard" will guide users through post-install steps:
+  adding new hosts, storage devices or services.
+* NFS: the Dashboard now allows users to fully manage all NFS exports from a single place.
+* New mgr module (feedback): users can quickly report Ceph tracker issues
+  or suggestions directly from the Dashboard or the CLI.
+* New "Message of the Day": cluster admins can publish a custom message in a banner.
+* Cephadm integration improvements:
+ * Host management: maintenance, specs and labelling,
+ * Service logs,
+ * Daemon management (start, stop, restart, reload),
+ * New services supported: ingress (HAProxy) and SNMP-gateway.
+* Monitoring and alerting:
+ * 43 new alerts have been added (totalling 68) improving observability of events affecting:
+   cluster health, monitors, storage devices, PGs and CephFS.
+ * Alerts can now be sent externally as SNMP traps via the new SNMP gateway service
+   (the MIB is provided).
+ * Improved integrated full/nearfull event notifications.
+ * Grafana Dashboards now use grafonnet format (though they're still available
+   in JSON format).
+ * Stack update: images for monitoring containers have been updated.
+   Grafana 8.3.5, Prometheus 2.33.4, Alertmanager 0.23.0 and Node Exporter 1.3.1.
+   This reduced exposure to several Grafana vulnerabilities (CVE-2021-43798,
+   CVE-2021-39226, CVE-2021-43798,  CVE-2020-29510, CVE-2020-29511).
```

Comment:
@ceph/dashboard please, have a look at this and check if there's something missing/wrong. Thanks!

Reply:
@ceph/dashboard On Host management, probably we can include the drain too and also the support for multi host creation with pattern.

And in service management ability to edit a service.
(Also not sure if its worthy to mention that we are using angular 12 now)

Reply:
Done! Thanks @nizamial09 ! I didn't mention the Angular 12 upgrade since it's less user-impacting than the Grafana/Prom/etc.

Reply:
make sense, thank you @epuertat And the drain and multi host thing?